src/modules/tests/test-data/cat-c/vehicle-checks/__tests__/vehicle-checks.cat-c.selector.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { vehicleChecksExist } from '../vehicle-checks.cat-c.selector';
+
+describe('Vehicle Checks Selector Cat BE', () => {
+
+  describe('vehicleChecksExist', () => {
+    it('should return false if there are no vehicle checks entered', ()  => {
+      const emptyVehicleChecks: CatBEUniqueTypes.VehicleChecks = { showMeQuestions: [{}], tellMeQuestions: [{}] };
```

Comment:
CatBEUniqueTypes instead of CatCUniqueTypes

Reply:
(multiple occurancs)

Reply:
changed it